wrapanapi/openstack.py

```diff
@@ -114,14 +116,40 @@ def api(self):
     @property
     def kapi(self):
         if not self._kapi:
-            self._kapi = oskclient.Client(username=self.username,
-                                          password=self.password,
-                                          tenant_name=self.tenant,
-                                          auth_url=self.auth_url,
-                                          insecure=True)
+            if self.keystone_version == 2:
+                from keystoneclient.v2_0.client import Client
+                self._kapi = Client(username=self.username,
+                                    password=self.password,
+                                    tenant_name=self.tenant,
+                                    auth_url=self.auth_url,
+                                    insecure=True)
+
+            if self.keystone_version == 3:
```

Comment:
This should be an elif. And at the end of the if block an else with exception changes.

Reply:
I moved it into __init__